index.js

```diff
@@ -37,7 +37,9 @@ const schema = {
     },
     headers: {
       type: 'object',
-      additionalProperties: true
+      additionalProperties: {
+        type: 'string'
```

Comment:
I'm not sure we should throw an error to users that set a number or simply `null`
We should serialize the object in the `lib/request.js` instead because, for example, a `date` object will be the ISO string

Reply:
Hi @Eomm, thanks for the review!
I thought about it, but serialize the input headers object could bring to other unexpected behaviour. For example, how we should serialize the `null` value, to string `"null"` or to empty string or should we remove the key? And a date object, should we serialize to iso string or to unix timestamp, or another format?
Being this library for testing and debugging purpose, I think users should have full control of the input, warning them in case of incorrect use